cluster-endpoints/src/grpc/gprc_accounts_streaming.rs

```diff
@@ -0,0 +1,215 @@
+                                            },
+                                        )),
+                                    }
+                                }
+                            })
+                            .collect_vec()
+                    } else {
+                        vec![]
+                    };
+                    subscribe_accounts.insert(
+                        format!("accounts_{}", program_id),
+                        SubscribeRequestFilterAccounts {
+                            account: vec![],
+                            owner: vec![program_id.clone()],
+                            filters,
+                        },
+                    );
+                }
+            }
+
+            let mut stream = client
+                .subscribe_once(
+                    Default::default(),
+                    subscribe_accounts,
+                    Default::default(),
+                    Default::default(),
+                    Default::default(),
+                    Default::default(),
+                    Some(commitment),
+                    Default::default(),
+                    None,
+                )
+                .await?;
+
+            while let Some(message) = stream.next().await {
+                let Ok(message) = message else {
+                    continue;
+                };
+
+                let Some(update) = message.update_oneof else {
+                    continue;
+                };
+
+                match update {
+                    UpdateOneof::Account(account) => {
+                        if let Some(account_data) = account.account {
+                            let account_pk_bytes: [u8; 32] = account_data
+                                .pubkey
+                                .try_into()
+                                .expect("pubkey should be deserializable");
+                            let owner: [u8; 32] = account_data
+                                .owner
+                                .try_into()
+                                .expect("owner pubkey should be deserializable");
+                            let notification = AccountNotificationMessage {
+                                data: AccountData {
+                                    pubkey: Pubkey::new_from_array(account_pk_bytes),
+                                    account: Account {
+                                        lamports: account_data.lamports,
+                                        data: account_data.data,
+                                        owner: Pubkey::new_from_array(owner),
+                                        executable: account_data.executable,
+                                        rent_epoch: account_data.rent_epoch,
+                                    },
+                                    updated_slot: account.slot,
+                                },
+                                commitment: Commitment::Confirmed,
```

Comment:
pls mention the code part that implies this confirmation level in a comment 
i.e.

> // we only get account changes from geyser subscription configured for commitment level CONFIRMED in method foo_bar()